Library.Console/Author.cs

```diff
@@ -9,11 +9,42 @@ public class Author
     public int Age { get; private set; }
 
     public HashSet<Book> Books { get; private set; } = new();
+    // fkc list, we dont wanna duplicates
+    // Decide that we cant remove book from author
 
+    public bool CheckBook(Book book) => Books.Where(e => e.ISBN == book.ISBN).Any();
+
+    public void AddBookToAuthor(Book book)
```

Comment:
Jak HashSet to docsy i sprawdzenie co robi metoda Add i jak można jej użyć, żeby nie dopuścić do dodanie do tego autora tej samej książki

Reply:
Sprawdzone poprawione